mautic/mautic
# Convert charset to utf8mb4 to resolve invalid character issues

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | No
| New feature/enhancement? (use the a.x branch)      | Yes
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | No 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Older instances that were created before the MySQL8 upgrades are using an outdated format. By migrating all tables issues like character encoding will be resolved.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make sure your table does not have `utf8mb4` charset 
3. Insert some Japanese character in text field or description field.
4. it should be saved and rendered properly
5. search filter should work fine.
